doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+- Para-Virtualization method (Hardware support).
+
+**Figure 3** below shows some of the relevant IO Virtualization techniques.
+
+<p align="center"><img src="./figures/tech_iov.png" alt="scope" title="Document Types" width="70%"/></p>
+<p align="center"><b>Figure 3:</b> Relevant IO Virtualization Techniques</p>
+
+<a name="4.1.1"></a>
+#### 4.1.1 Para-virtualization method (software only)
+
+This is the preferred method of IO virtualization as it provides flexibility and full abstraction of workloads from the underlying infrastructure. It usually relies on standard IO interfaces that are implemented in software.
+For Networking, there are two common networking interfaces used: virtio-net for KVM/QEMU and VMXNET for VMware.
+
+Using a standard interface for IO means that workload doesn't need to run any proprietary software drivers for specific hardware vendors and the implementation of that workload is completely agnostics of the hardware used.
+
+**Figure 4** below shows the typical components of a para-virtualized interface: 
+- frontEnd driver: The frontEnd driver is an off-the-shelf driver that runs on the workload.
+- backEnd driver: runs on the Hypervisor and is responsible of bridging standard communications coming from applications to a hardware specific ones. 
+
+This nature of this disaggregation is what gives the para-virtualized interfaces the flexibility that makes them favourable in a virtualized environment.
+
+The downside of para-virtualization interfaces is the involvement of the hypervisor which may introduce latency and jitter that can impact the performance.
+
+<p align="center"><img src="./figures/tech_virtio.png" alt="virtio" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 4:</b> Para-Virtualized interface components (software only).</p>
+
+<a name="4.1.2"></a>
+#### 4.1.2 Direct assignment with Virtual Technology.
+
+Direct assignment (or pass-through) is when an IO device is directly assigned to a workload by-passing the hypervisor. Direct assignment is supported in Intel platforms (using VT-d technology) and by AMD platforms (using AMD-V technology) as shown in **Figure 5**.
+
+Once an IO device is directly assigned to a workload, that workload will then have an exclusive access to that device and no other entities (including the hypervisor) can access it.
+
+This method provides better performance than the para-virtualized one as no hypervisor is involved but provides less flexibility and less portability.
+
+Having an IO device directly assigned to a workload means that the workload needs to run vendor specific drivers and libraries to be able to access that device which makes the workload less portable and dependent on a specific hardware type from a specific vendor which is not aligned with the overall strategy and goals of CNTT and hence this method of IO Virtualization should not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
```

Comment:
Requiring vendor-specific drivers is a violation of an explicit CNTT principle. As written ("should not be used") it implies a loophole. Consider something more enforceable (e.g. may not be used, or some such). Possibly worthy of discussion.

Reply:
Resolved.